### PR TITLE
[IMP] mail: set there is no message text in italic

### PR DESCRIPTION
**Pull request by jgi-odoo**
task-id:2343510

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jgi-odoo** [comment]
@alexkuhn @seb-odoo

File: addons/mail/static/src/components/message_list/message_list.scss [hunk 1]
```diff
@@ -28,6 +28,7 @@
     justify-content: center;
     padding: 20px;
     line-height: 2.5rem;
+    font-style: italic;
```
Comment (seb-odoo) [inline addons/mail/static/src/components/message_list/message_list.scss:31]: Should be moved to the Style category, not Layout.

I'll test to see how it looks.

Comment (seb-odoo) [reply addons/mail/static/src/components/message_list/message_list.scss:31]: Would be nice to have an enterprise runbot to test there as well, I think the main "issue" was when the chatter is on the right

Comment (jgi-odoo) [reply addons/mail/static/src/components/message_list/message_list.scss]: Done

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**